src/Microsoft.TestPlatform.Common/DataCollection/DataCollectionManager.cs

```diff
@@ -628,5 +658,40 @@ private void LogAttachments(List<AttachmentSet> attachmentSets)
                 }
             }
         }
+
+        /// <summary>
+        /// Initializes the Data Collector Cache.
+        /// </summary>
+        /// <param name="extensionsPath">Extensions path.</param>
+        private void InitializeDataCollectorCache(string extensionsPath)
+        {
+            var extpaths = string.IsNullOrWhiteSpace(extensionsPath) ? new List<string>() : extensionsPath.Split(new[] { ';' }, StringSplitOptions.RemoveEmptyEntries).ToList();
+
+            // add current directory and extensions folder path. 
+            // todo : remove this once extensions path starts coming from vstest.console.
+            var currentDir = Path.GetDirectoryName(typeof(DataCollectionManager).GetTypeInfo().Assembly.Location);
```

Comment:
Create an issue for this please? We need to bring the support for `ExtensionPath` for all other extensibility types and remove the hardcoding for `Extensions` directory. /cc: @Faizan2304